### DISCUSSION TITLE
textarea组件，在有大段文字时，从中间长按删除键，光标会移至最后面

### DISCUSSION THREAD

**Discussion by cmdparkour** [NervJS/taro#11160] [Q & A]
textarea组件，在有大段文字时，从中间长按删除键，光标会移至最后面。
大家有遇到这种情况没
![image](https://user-images.githubusercontent.com/35911743/150323915-c4f3e782-8dbc-4e55-8f9a-35e660bdd0d5.png)

**Comment by cmdparkour** [comment]
技术栈是：react
taro版本是：3.4.0
看了一下源码，不知道是不是和handleInput这个函数有关，看到里面是有一个cusor: e.detail.value.length的一个设置

**Comment by Josie-ljw** [comment]
我也遇到了这个问题，问题表现是：
一段文字，移动光标到中间，无法连续删除文字，删除完一个字之后，光标就会移动到最后

**Comment by Ke-Kou** [comment]
这个问题现在解决了吗

**Comment by 17679336673** [comment]
我也遇到了，react 3.3.3，还提bug给我，麻了🃏

**Comment by siriuswh0327** [comment]
同遇到，有issue么？

**Comment by redpandaex** [comment]
2026年了,这个问题有解决吗?

**Comment by peijunlei** [comment]
cursor=-1试试
